integration/toxi/toxi_spec.rb

```diff
@@ -78,22 +77,89 @@ def warm_up
     # Connect (the pool is lazy)
     Sharded.where(id: 1).first
     errors = 0
+    ok = 0
```

Comment:
Might want a `Concurrent::AtomicFixnum` here?

Reply:
Don't need one, these are not parallel.